<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>anapencere</class>
 <widget class="QMainWindow" name="anapencere">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>anapencere</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>451</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andina Demo</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 20pt &quot;Andina Demo&quot;;
color:#f4cccc</string>
    </property>
    <property name="text">
     <string>Yöre Yöre, Yiye Yiye</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>581</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QTableWidget{background: white;}
QTableWidget::item{background: white;  border: 2px solid #f4cccc;}
QTableWidget::item::pressed{background:  #e06666;}
QTableWidget::item::hover{background:  #e06666;}
QHeaderView::section { background-color:#f4cccc; color:#e06666; font-weight:800}
QTableView QTableCornerButton::section{ background-color:#f4cccc }
QTableWidget::item:selected{ background-color: #e06666 }
</string>
    </property>
    <column>
     <property name="text">
      <string>1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>5</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>581</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>A Year Without Rain</family>
         <pointsize>14</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: black;
font: 14pt &quot;A Year Without Rain&quot;;</string>
       </property>
       <property name="text">
        <string>Yemek Listesi</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_panel">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
font: 500 10pt &quot;Arimo Medium&quot;;
font-weight:400;
color: white;
background: #f4cccc;
color:white;
border: 2px solid #f4cccc;
padding: 5px;}
QPushButton:hover {
background-color: #ea9999;
border: 2px solid #ea9999;
}
QPushButton:pressed {
background-color: #e06666;
border: 2px solid #e06666;
}</string>
       </property>
       <property name="text">
        <string>Yemek Paneli Giriş</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
